src/index/StringSortComparator.h

```diff
@@ -205,6 +207,23 @@ class LocaleManager {
     return {numCodepoints, std::string(sp.data(), i)};
   }
 
+  /**
+   * @brief Normalize a Utf8 string to a canonical representation.
+   * Maps e.g. single codepoint é and e + accent aigu to single codepoint é by
+   * applying the UNICODE NFC (Normalization form C) This is independent from
+   * the locale
+   * @param input The String to be normalized. Must be UTF-8 encoded
+   * @return The NFC canonical form of NFC in UTF-8 encoding.
+   */
+  std::string normalizeUtf8(std::string_view input) const {
```

Comment:
I opened Issue #313 for this so I don't forget it. I do not want to do this while I still have PRs open that modify it to not get into rebasing hell. Otherwise I agree.